src/main/java/bio/terra/common/DaoUtils.java

```diff
@@ -49,7 +49,9 @@ public static void addFilterClause(
     if (!StringUtils.isEmpty(filter)) {
       params.addValue("filter", DaoUtils.escapeFilter(filter));
       clauses.add(
-          String.format(" (%s.name ILIKE :filter OR %s.description ILIKE :filter) ", table, table));
+          String.format(
+              " (%s.id::text ILIKE :filter OR %s.name ILIKE :filter OR %s.description ILIKE :filter) ",
+              table, table, table));
```

Comment:
Oh, neat - I don't think I'm familiar with the "::" notation as a cast, but definitely a concise solution! Nice!

Reply:
Yeah! I thought postgres would handle that automatically but it fails without doing this cast 🤔